package.json

```diff
@@ -53,7 +53,9 @@
     "winston-elasticsearch": "^0.5.0",
     "xml2js": "^0.4.16",
     "youtube-api": "^2.0.2",
-    "youtube-dl": "^1.11.1"
+    "youtube-dl": "^1.11.1",
+    "lodash": "^4.17.4",
+    "numeral": "^2.0.6"
```

Comment:
Is there a valid reason for using this dependency as well, or could it be accomplished with basic JavaScript too?

Reply:
Not sure what the project's philosophy is, but basic javascript is kind of unreliable in the longterm, this simplifies code and if used consistently makes it less brittle. Definitely can be removed though.